eng/Version.Details.xml

```diff
@@ -30,13 +30,21 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>ee39cd1573dbb8011f343e1037af51d4fc00a747</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App" Version="5.0.0-rc.2.20479.6">
-      <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>c95b3204bc3790e797cf8787481898e5473049d9</Sha>
+    <Dependency Name="Microsoft.AspNetCore.App.Ref.Internal" Version="5.0.0-rtm.20517.7">
+      <Uri>https://github.com/dotnet/aspnetcore</Uri>
+      <Sha>c7d42cf37bfc928bfae6c4ddaff0749fbb2fb9f2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0-rc.2.20475.17">
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="5.0.0">
```

Comment:
For ASP, do we need to go back to RC2 for this to work?

Reply:
Actually, please just make these consistent with the values in Versions.props (sha and version) please
For ASP 5.0.0-rc.2.20475.17 is 25bd01f17d5f01fc78bea5eb5308b2c0d79e2889 for both this and the one above.